.github/CODEOWNERS

```diff
@@ -0,0 +1,428 @@
+/mlir/**/SPIRV/ @llvm/pr-subscribers-mlir-spirv
+
+# mlir-shape
+/mlir/**/Shape/ @llvm/pr-subscribers-mlir-shape
+
+# mlir-sparse
+/mlir/**/SparseTensor/ @llvm/pr-subscribers-mlir-sparse
+
+# mlir-tensor
+/mlir/**/Tensor/ @llvm/pr-subscribers-mlir-tensor
+
+# mlir-tosa
+/mlir/**/Tosa/ @llvm/pr-subscribers-mlir-tosa
+
+# mlir-ub
+/mlir/**/UB/ @llvm/pr-subscribers-mlir-ub
+
+# mlir-vector
+/mlir/**/*Vector @llvm/pr-subscribers-mlir-vector
+
+# mlir-execution-engine
+/mlir/ExecutionEngine @llvm/pr-subscribers-mlir-execution-engine
+
+# coroutines
+/clang/docs/DebuggingCoroutines.rst @llvm/pr-subscribers-coroutines
+/clang/lib/Sema/SemaCoroutine.cpp @llvm/pr-subscribers-coroutines
+/clang/lib/CodeGen/CGCoroutine.cpp @llvm/pr-subscribers-coroutines
+/clang/test/CodeGenCoroutines/ @llvm/pr-subscribers-coroutines
+/llvm/docs/Coroutines.rst @llvm/pr-subscribers-coroutines
+/llvm/include//llvm/Transforms/Coroutines/ @llvm/pr-subscribers-coroutines
+/llvm/lib/Transforms/Coroutines/ @llvm/pr-subscribers-coroutines
+/llvm/test/Transforms/Coroutines/* @llvm/pr-subscribers-coroutines
+
+# modules
+/clang/docs/StandardCPlusPlusModules.rst @llvm/pr-subscribers-modules
+/clang/include/clang/AST/AbstractBasicReader.h @llvm/pr-subscribers-modules
+/clang/include/clang/AST/AbstractBasicWriter.h @llvm/pr-subscribers-modules
+/clang/include/clang/AST/AbstractTypeReader.h @llvm/pr-subscribers-modules
+/clang/include/clang/AST/AbstractTypeWriter.h @llvm/pr-subscribers-modules
+/clang/include/clang/AST/PropertiesBase.td @llvm/pr-subscribers-modules
+/clang/include/clang/AST/ODRHash.h @llvm/pr-subscribers-modules
+/clang/include/clang/AST/TypeProperties.td @llvm/pr-subscribers-modules
+/clang/include/clang/Basic/Module.h @llvm/pr-subscribers-modules
+/clang/include/clang/Frontend/PrecompiledPreamble.h @llvm/pr-subscribers-modules
+/clang/include/clang/Lex/ModuleLoader.h @llvm/pr-subscribers-modules
+/clang/include/clang/Lex/ModuleMap.h @llvm/pr-subscribers-modules
+/clang/include/clang/Serialization/ @llvm/pr-subscribers-modules
+/clang/lib/AST/ODRHash.cpp @llvm/pr-subscribers-modules
+/clang/lib/AST/StmtProfile.cpp @llvm/pr-subscribers-modules
+/clang/lib/Basic/Module.cpp @llvm/pr-subscribers-modules
+/clang/lib/Frontend/ModuleDependencyCollector.cpp @llvm/pr-subscribers-modules
+/clang/lib/Frontend/PrecompiledPreamble.cpp @llvm/pr-subscribers-modules
+/clang/lib/Lex/ModuleMap.cpp @llvm/pr-subscribers-modules
+/clang/lib/Sema/SemaModule.cpp @llvm/pr-subscribers-modules
+/clang/lib/Serialization/ @llvm/pr-subscribers-modules
+/clang/test/CXX/module/ @llvm/pr-subscribers-modules
+/clang/test/Modules/ @llvm/pr-subscribers-modules
+/clang/unittests/Serialization/* @llvm/pr-subscribers-modules
+
+# clang-tidy
+/clang-tools-extra/clang-tidy/ @llvm/pr-subscribers-clang-tidy
+/clang-tools-extra/docs/clang-tidy/ @llvm/pr-subscribers-clang-tidy
+/clang-tools-extra/test/clang-tidy/ @llvm/pr-subscribers-clang-tidy
+
+# llvm-mca
+/llvm/tools//llvm-mca/ @llvm/pr-subscribers-llvm-mca
```

Comment:
`//` should be `/*/`

Worth grepping other occurrences of `//`

Reply:
Are you sure // should be /*/ in this case?

Reply:
OK. I believe it should be
```
/llvm/tools/llvm-mca/
/llvm/test/tools/llvm-mca/
```

`//` in gitignore does not match anything.